Makefile

```diff
@@ -254,14 +257,14 @@ PACKAGE_OFFLINE_PARA=-zcvf harbor-offline-installer-$(PKGVERSIONTAG).tgz \
 					$(HARBORPKG)/prepare \
 					$(HARBORPKG)/LICENSE $(HARBORPKG)/install.sh \
 					$(HARBORPKG)/common.sh \
-					$(HARBORPKG)/harbor.yml
+					$(HARBORPKG)/harbor.yml.tmpl
```

Comment:
we need to add the `harbor.yml.tmpl` existence validation in `prepare`

Reply:
added check for harbor.yml in prepare file